packages/react-native-win32/.flowconfig

```diff
@@ -29,9 +30,11 @@
 ; Ignore react-native files in node_modules since they are copied into project root
 .*/node_modules/react-native/.*
 
-; Ignore react-native-windows' target folder. Flow is not compiled with long path support and after running unittests these folders have long paths
+; Ignore react-native-windows' build output. Flow is not compiled with long path support and after running unittests these folders have long paths
```

Comment:
At some point we should look at restricting flow a bit so it doesn't do more work than it needs to